docs/publish/publish-add-in-vs-code.md

```diff
@@ -14,9 +14,37 @@ This article describes how to publish an Office Add-in that you created using th
 
 ## Publishing an add-in for other users to access
 
-[!include[instructions for publishing an Office Add-in](../includes/publish-add-in.md)]
+An Office Add-in consists of a web application and a manifest file. The web application defines the add-in's user interface and functionality, while the manifest specifies the location of the web application and defines settings and capabilities of the add-in.
+
+While you're developing your add-in, you can run the add-in on your local web server (`localhost`), but when you're ready to publish it for other users to access, you'll need to deploy the web application and update the manifest to specify the URL of the deployed application.
+
+When your add-in is working as desired, you can publish it directly through Visual Studio Code using the Azure Storage extension.
+
+## Using Visual Studio Code to publish
+
+1. Open your project from its root folder in VS (Visual Studio) Code.
+2. From the Extensions view in VS Code, search for the Azure Storage extension and install it.
+3. Once installed, an Azure icon will be added to the Activity Bar. Select it to access the extension. If your Activity Bar is hidden, you won't be able to access the extension. Show the Activity Bar by selecting **View > Appearance > Show Activity Bar**.
+4. When in the extension, sign in to your Azure account by selecting **Sign in to Azure**. You can also create an Azure account if you don't already have one by selecting **Create a free Azure account**. Follow the provided steps to set up your account.
+5. Once you have signed in to your Azure account, you'll see your Azure storage accounts appear in the extension. If you don't already have a storage account, you'll need to create one using the **Create new storage account** option. Name your storage account a globally unique name, using only 'a-z' and '0-9'. Note that by default, this creates a storage account and a resource group with the same name. It automatically puts the storage account in West US. This can be adjusted online through [your Azure account](https://portal.azure.com/).
```

Comment:
```suggestion
5. Once you have signed in to your Azure account, you'll see your Azure storage accounts appear in the extension. If you don't already have a storage account, create one using the **Create new storage account** option. Name your storage account a globally unique name, using only 'a-z' and '0-9'. Note that by default, this creates a storage account and a resource group with the same name. It automatically puts the storage account in West US. This can be adjusted online through [your Azure account](https://portal.azure.com/).
```

Reply:
I can't see a difference here so just marking as resolved.